src/bots/mascot-bot.js

```diff
@@ -59,6 +80,51 @@ class MascotBot extends SlackBot {
     }
   }
 
+  postMessage(...args) {
+    if (this.debug) {
+      this.log('API limited in debug mode', true);
```

Comment:
Yes! I wanted to do that but I guess I got hasty and forgot to update the log message for it.